src/c++/libos/lwip/lwip_queue.hh

```diff
@@ -66,8 +67,11 @@ class lwip_queue : public io_queue {
     private: static std::map<lwip_4tuple, std::queue<dmtr_sgarray_t> *> our_recv_queues;
     private: static std::unordered_map<std::string, struct in_addr> our_mac_to_ip_table;
     private: static std::unordered_map<in_addr_t, struct rte_ether_addr> our_ip_to_mac_table;
-    private: static uint16_t my_app_port; //FIXME this could/should be a list
-    public: static void set_app_port(uint16_t port) { my_app_port = port; }
+
+    // ???: Should this be static? (all queues share the same my_app_ports instance?)
```

Comment:
The reason why sharing the port list across queues is needed at the moment is because a rx_burst can dequeue packets destined to other queues. If we multithread this stack, each net worker should have its own list of ports.

Reply:
Yeah, makes sense. Adding a note to that effect in the code.